physical/raft/raft_autopilot.go

```diff
@@ -211,6 +211,86 @@ type FollowerState struct {
 	RedundancyZone  string
 }
 
+// Copy returns a copy of the follower state.
+// This copy uses the same pointer to the IsDead
```

Comment:
After reading the code below, where this is used in `KnownServers()`, I see what you're getting at now. If `IsDead` changes between when we call `Copy()` and when we check that field, we want to know that, instead of using a potentially stale value, so this is indeed what we actually want.

My one concern with this approach is that someone repurposes `Copy()` later on for something else, especially since it's a public method, and doesn't pay attention to these comments, and thinks they're getting a real copy when they're actually getting a 90% copy, and then shenanigans ensue. Any harm in making this an internal method instead, and maybe changing the name to `partialCopy()` or something like that? I might be overly paranoid here, and if you think so, it's fine to skip this.

Reply:
makes sense! I changed it to partialCopy